sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetPartitionDiscoverySuite.scala

```diff
@@ -249,6 +249,11 @@ class ParquetPartitionDiscoverySuite extends QueryTest with ParquetTest with Sha
           true,
           rootPaths,
           timeZoneId)
+      assert(actualSpec.partitionColumns === spec.partitionColumns)
+      assert(actualSpec.partitions.length === spec.partitions.length)
+      actualSpec.partitions.zip(spec.partitions).foreach { case (actual, expected) =>
+        assert(actual === expected)
+      }
```

Comment:
just for my own understanding -- these added asserts are just to improve failure msgs, right?  They would have all been covered by the `assert(actualSpec === spec)` below anywyay, right?

Reply:
Yes, it is. It was difficult for me to find which one was different.